.snyk

```diff
@@ -0,0 +1,17 @@
+# References:
+# https://docs.snyk.io/scan-applications/snyk-code/using-snyk-code-from-the-cli/excluding-directories-and-files-from-the-snyk-code-cli-test
+# https://docs.snyk.io/snyk-cli/commands/ignore
+exclude:
+  global:
+    - "cnf-tests/submodules/metallb-operator/**"
+    - "cnf-tests/submodules/sriov-network-operator/**"
```

Comment:
if it works, I'd go as far as
```
- "cnf-tests/submodules/*/vendor/**"
```

Reply:
there are two things
1. we are ignoring _everything_ in the metallb-operator and sriov-network-operator submodels. Is that what we want to do?
2. do we want to ignore all submodules vendors?

Reply:
In my mind we only care about what's related to the cnf-tests image that we ship to customers.
and metallb+sriov are not part of it.

And in the NTO submodule vendor I don't want to ignore all because for example we compile and include ginkgo lib in the image.

Reply:
whereas on different repos they don't care about the ginkgo lib which is only used for testing and not shipped as part of the product binary. does that makes sense?